wrappers/winbase.go

```diff
@@ -569,6 +574,16 @@ func CreateProcess(applicationName *uint16, commandLine *uint16, processAttribut
 	return nil
 }
 
+func SetProcessShutdownParameters(level, flags uint32) bool {
```

Comment:
Should return an `error` like the other functions here, rather than a `bool`.

Reply:
Of course. Done.